There's no way to build wireless extensions out of tree
so remove the Kconfig options for them.

--- a/net/wireless/Kconfig
+++ b/net/wireless/Kconfig
@@ -1,21 +1,3 @@
-config WIRELESS_EXT
-	bool
-
-config WEXT_CORE
-	def_bool y
-	depends on CFG80211_WEXT || WIRELESS_EXT
-
-config WEXT_PROC
-	def_bool y
-	depends on PROC_FS
-	depends on WEXT_CORE
-
-config WEXT_SPY
-	bool
-
-config WEXT_PRIV
-	bool
-
 config CFG80211
 	tristate "cfg80211 - wireless configuration API"
 	depends on RFKILL || !RFKILL
@@ -177,7 +159,7 @@ config CFG80211_INTERNAL_REGDB
 config CFG80211_WEXT
 	bool "cfg80211 wireless extensions compatibility"
 	depends on CFG80211
-	select WEXT_CORE
+	depends on WEXT_CORE
 	help
 	  Enable this option if you need old userspace for wireless
 	  extensions with cfg80211-based drivers.
